Update and improve the API landing page, add more topics (#1376)
aiven/devportal: docs/tools/api/concepts/tokens.rst

Work with API authorization tokens
==================================

> The Aiven API uses tokens for authentication. In this article, we will cover some of the key considerations for working with API tokens.

Reviewer: ```suggestion
The Aiven API uses tokens for authentication. In this article, we cover some of the key considerations when working with API tokens.
```

Limit token expiry
------------------

> Think about how long the token needs to be valid for; this will vary with the situation that the token is used for.

Reviewer: ```suggestion
Think about how long the token needs to be valid for; this varies depending on what the token is used for.
```
